includes/Integrations/Core_Themes_Support.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Class Core_Themes_Support
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
+
+use Google\Web_Stories\Customizer;
+use Google\Web_Stories\Stories_Renderer\Renderer;
+use Google\Web_Stories\Traits\Assets;
+use function Google\Web_Stories\render_theme_stories;
+
+/**
+ * Class Core_Themes_Support.
+ */
+class Core_Themes_Support {
+	use Assets;
+
+	/**
+	 * Default array of core themes to add support to.
+	 *
+	 * @var array
+	 */
+	protected static $supported_themes = [
+		'twentytwentyone',
+		'twentytwenty',
+		'twentynineteen',
+		'twentyseventeen',
+		'twentysixteen',
+		'twentyfifteen',
+		'twentyfourteen',
+		'twentythirteen',
+		'twentytwelve',
+		'twentyeleven',
+		'twentyten',
+	];
+
+	/**
+	 * Adds theme support for Web Stories.
+	 *
+	 * This will enable add_theme_support with predefined
+	 * options supported themes.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @return void
+	 */
+	public function extend_theme_support() {
+		add_theme_support( 'web-stories' );
+	}
+
+	/**
+	 * Embed Webstories.
+	 *
+	 * Embeds web stories with default customizer settings.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @return void
+	 */
+	public function embed_web_stories() {
+		if ( is_readable( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/css/web-stories-theme-style-' . get_stylesheet() . '.css' ) ) {
+			$this->enqueue_style( 'web-stories-theme-style-' . get_stylesheet(), [ Renderer::STYLE_HANDLE ] );
+		}
```

Comment:
```suggestion
		$stylesheet = get_stylesheet();
		if ( is_readable( sprintf( '%s/assets/css/web-stories-theme-style-%s.css', WEBSTORIES_PLUGIN_DIR_PATH, $stylesheet ) ) {
			$this->enqueue_style( 'web-stories-theme-style-' . $stylesheet, [ Renderer::STYLE_HANDLE ] );
		}
```

This feels a little cleaner here.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6543/commits/945b8428cd8d007795dd1a81360251ed4599e1f2